tests/Integration/Subscriber/ExpiredCachePurgeSubscriber/TestCleanCacheScheduledEvent.php

```diff
@@ -27,8 +30,6 @@ public function testShouldNotCleanScheduledEventWhenValuesAreTheSame() {
 				'purge_cron_unit'     => 'HOUR_IN_SECONDS',
 			]
 		);
-
-		$this->assertTrue( true ); // Prevent "risky" warning.
```

Comment:
Why remove this?

Reply:
It's not testing anything. [`Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration`](https://github.com/wp-media/phpunit/blob/master/Integration/TestCase.php#L12), included in the base `TestCase`, eliminates needing the "risky" warning.

Reply:
Removing it won't trigger an error saying that the test does no assert?

Reply:
Nope. Why? There is an assertion in the test.

```php
Functions\expect( 'wp_clear_scheduled_hook' )->never();
```

Reply:
I've seen several times where this would not be considered as an assertion, and the error would trigger.
Maybe I did something wrong at the time, or maybe it was a bug in an old version of BrainMonkey/Mockery (I'm not sure but I think I remember reading something about this). Also, I don't think I am the author of this, so I'm guessing I'm not the only one experiencing this 🤔 
Anyway, let's keep them removed, and see if there are error messages.
Thanks Tonya.

Reply:
Before the `Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration` was not used in the base Integration Test Case. Without it, an error would trigger with the "risky" warning.

Reply:
OK, understood, thanks.